official/resnet/keras/keras_imagenet_main.py

```diff
@@ -193,6 +193,10 @@ def main(_):
 
 if __name__ == '__main__':
   tf.compat.v1.logging.set_verbosity(tf.compat.v1.logging.INFO)
+  # Use BFC allocator with limited memory on CPU as a workaround for memory
+  # spikes in eager mode.
```

Comment:
Also leave a TODO for you to remove this?

Reply:
Done